sdk/src/main/java/com/phunware/parkassist/ParkAssistSDK.java

```diff
@@ -265,22 +252,16 @@ public void getVehicleThumbnail(Location location, String uuid, final Callback<B
         Map<String, String> params = new HashMap<>();
         params.put(PARAMS_LATITUDE, latitude);
         params.put(PARAMS_LONGITUDE, longitude);
-        ParkAssistHttpClient.getImage(generateGetURL(String.format(THUMBNAIL_ENDPOINT_FORMAT, uuid), params),
-                new BinaryHttpResponseHandler() {
+        httpClient.getImage(generateGetURL(String.format(THUMBNAIL_ENDPOINT_FORMAT, uuid), params),
+                new ParkAssistNetworkingInterface.ParkAssistImageResponseInterface() {
                     @Override
-                    public String[] getAllowedContentTypes() {
-                        String[] contentTypes = {"image/jpeg", "text/html; charset=utf-8"};
-                        return contentTypes;
+                    public void onSuccess(byte[] imageData) {
+                        Bitmap image = BitmapFactory.decodeByteArray(imageData, 0, imageData.length);
+                        callback.onSuccess(image);
```

Comment:
how does bitmapfactory act with corrupt bitmap data?  What should we do in case of OOM exception?